CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- [#102] Redesign dogu-starting-page
```

Comment:
![grafik](https://github.com/user-attachments/assets/bb6b86aa-aaa1-4e6d-b8ba-0c6ff66ec32b)

uh... one more thing. Since we now actively translate UI text to german, I would expect this kind of change also in the changelog
```suggestion
- [#102] Redesign dogu-starting-page
- Change UI language to german
```